packages/salesforcedx-vscode-core/src/telemetry/telemetry.ts

```diff
@@ -127,20 +133,25 @@ export class TelemetryService {
 
   public sendCommandEvent(
     commandName?: string,
-    hrstart?: [number, number]
+    hrstart?: [number, number],
+    additionalData?: any
```

Comment:
Doing it this way allows us to expand metrics on command execution without touching the interface that often.